test/test_credential.js

```diff
@@ -0,0 +1,245 @@
+const _deploy_contracts = require('../migrations/2_deploy_contracts');
+const truffleAssert = require('truffle-assertions');
+const BigNumber = require('bignumber.js'); // npm install bignumber.js
+var assert = require('assert');
+
+const oneEth = new BigNumber(1000000000000000000); // 1 eth
+const zeroAddress = '0x0000000000000000000000000000000000000000';
+
+const toUnixTime = (year, month, day) => {
+  const date = new Date(Date.UTC(year, month - 1, day));
+  return Math.floor(date.getTime() / 1000);
+};
+
+const toDate = unixTimestamp => new Date(unixTimestamp * 1000);
+
+var Credential = artifacts.require('../contracts/Credential.sol');
+var Institution = artifacts.require('../contracts/Institution.sol');
+
+contract('Credential', function (accounts) {
+  before(async () => {
+    credentialInstance = await Credential.deployed();
+    institutionInstance = await Institution.deployed();
+  });
+
+  console.log('Testing Credential contract');
+
+  it('Add Credential', async () => {
+    // Create approved institution
+    let makeI1 = await institutionInstance.addInstitution(accounts[1], 'National University of Singapore');
```

Comment:
small thing but i think this can be const, we are not mutating it in the function. same for the others

Reply:
mm i see what u mean, changed!

Reply:
wait do you mean to change all the lets in this test file? since all arent mutated after creation actually
i initially put all of them as let coz thats what prof did in his labs and midterms HAHA